src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.cs

```diff
@@ -149,7 +149,7 @@ private void StartSearch(INavigateToCallback callback, string searchValue, IImmu
                 kinds,
                 _cancellationTokenSource.Token);
 
-            searcher.Search();
+            _ = searcher.SearchAsync();
```

Comment:
Doesn't this have an effect of silencing all exceptions?

Reply:
➡️ It's an intermediate step. VSTHRD110 deals with observing the result of asynchronous operations.